app/clients/hedera/client_test.go

```diff
@@ -0,0 +1,181 @@
+/*
+* Copyright 2021 LimeChain Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package hedera
+
+import (
+	"github.com/hashgraph/hedera-sdk-go/v2"
+	"github.com/limechain/hedera-eth-bridge-validator/app/model/transfer"
+	tc "github.com/limechain/hedera-eth-bridge-validator/test/test-config"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewNodeClient(t *testing.T) {
+	nodeClient := NewNodeClient(tc.TestConfig.Validator.Clients.Hedera)
+	assert.NotNil(t, nodeClient)
+}
+
+func TestGetClientForTestnet(t *testing.T) {
+	nodeClient := NewNodeClient(tc.TestConfig.Validator.Clients.Hedera)
+	client := nodeClient.GetClient()
+	assert.NotNil(t, client)
```

Comment:
Here you assert that the client is not nil. Are there any properties inside the client that differ it from the other 2 tests related to getting different types of clients?